pkg/controller/logstorage/secrets/secret_controller.go

```diff
@@ -154,6 +151,11 @@ func Add(mgr manager.Manager, opts options.AddOptions) error {
 	if err := utils.AddServiceWatch(c, render.LinseedServiceName, helper.InstallNamespace()); err != nil {
 		return fmt.Errorf("log-storage-secrets-controller failed to watch Service: %w", err)
 	}
+	if opts.MultiTenant {
+		if err = utils.AddSecretsWatchWithHandler(c, certificatemanagement.TenantCASecretName, "", &handler.EnqueueRequestForObject{}); err != nil {
```

Comment:
nit: Looks like we already have a convenience func that creates the handler: `utils.AddSecretsWatch`